package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "main": "Gruntfile.js",
+  "dependencies": {
+    "grunt": "^1.3.0",
+    "grunt-cli": "^1.3.2",
+    "grunt-contrib-qunit": "^4.0.0",
+    "phantomjs-prebuilt": "2.1.15"
+  },
+  "devDependencies": {},
```

Comment:
Note that I intentionally left this in. I'd rather be explicit in listing that there are no dev dependencies than have people guessing.